tests/manage/z_cluster/cluster_expansion/test_add_capacity_with_node_restart.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import logging
+import time
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework.testlib import ignore_leftovers, ManageTest, tier4
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources import pod as pod_helpers
+from ocs_ci.ocs.resources import storage_cluster
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.node import get_typed_nodes, wait_for_nodes_status
+
+
+@pytest.mark.parametrize(
+    argnames=["node_type", "num_of_nodes"],
+    argvalues=[
+        pytest.param(
+            *['worker', 1],
+            marks=pytest.mark.polarion_id("OCS-1313")
+        ),
+     ]
+)
+@pytest.mark.parametrize(
+    argnames=["workload_storageutilization_rbd"],
+    argvalues=[pytest.param(
+        (0.25, False, 120),
+        marks=pytest.mark.polarion_id("OCS-1313"),
```

Comment:
it's the same polarion ID as in line21. Can you please create a separate Polarion test case?